dags/atd_knack_flashing_beacons.py

```diff
@@ -0,0 +1,92 @@
+from datetime import datetime, timedelta
+from airflow.models import DAG
+from airflow.models import Variable
+from airflow.operators.docker_operator import DockerOperator
+from _slack_operators import task_fail_slack_alert
+
+default_args = {
+    "owner": "airflow",
+    "description": "Load flashing beacons (view_1563) records from Knack to Postgrest to AGOL and Socrata",  # noqa:E501
+    "depend_on_past": False,
+    "start_date": datetime(2021, 4, 14),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "retry_delay": timedelta(minutes=5),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+docker_image = "atddocker/atd-knack-services:production"
+
+# command args
+script_task_1 = "records_to_postgrest"
+script_task_2 = "records_to_socrata"
+script_task_3 = "records_to_agol"
+app_name = "data-tracker"
+container = "view_1563"
+env = "prod"
+
+# assemble env vars
+env_vars = Variable.get("atd_knack_services_postgrest", deserialize_json=True)
+atd_knack_auth = Variable.get("atd_knack_auth", deserialize_json=True)
+env_vars["KNACK_APP_ID"] = atd_knack_auth[app_name][env]["app_id"]
+env_vars["KNACK_API_KEY"] = atd_knack_auth[app_name][env]["api_key"]
+env_vars["SOCRATA_API_KEY_ID"] = Variable.get("atd_service_bot_socrata_api_key_id")
+env_vars["SOCRATA_API_KEY_SECRET"] = Variable.get(
+    "atd_service_bot_socrata_api_key_secret"
+)
+env_vars["SOCRATA_APP_TOKEN"] = Variable.get("atd_service_bot_socrata_app_token")
+env_vars["AGOL_USERNAME"] = Variable.get("agol_username")
+env_vars["AGOL_PASSWORD"] = Variable.get("agol_password")
+
+with DAG(
+    dag_id="atd_knack_flashing_beacons",
+    default_args=default_args,
+    schedule_interval="15 3 * * *",
```

Comment:
i'm too lazy to get on VPN to check Airflow: are you keeping these new DAGs spaced a few minutes apart from each other? 5 or 10 min should be fine.

Reply:
I used the time from the old configuration, I'm signing into airflow to make sure it doesnt overlap